src/less/skeleton/skeleton.less

```diff
@@ -0,0 +1,190 @@
+@import "../variables/variables.less";
+@import "../mixins/public/skeleton-mixins.less";
+
+@skeleton-text-border-radius: 3px;
+@skeleton-text-height-small: 16px;
+@skeleton-text-height-large: 24px;
+
+.skeleton {
+    container-name: skeleton-container;
+    container-type: inline-size;
+}
+
+.skeleton__avatar {
+    .skeleton-base(50%, var(--skeleton-height), var(--skeleton-height));
+}
+
+.skeleton__button {
+    .skeleton-base(20px, 40px);
```

Comment:
Can we use variables for this?

Reply:
I would prefer to keep it as is for now.  As this needs to be done from component level and inherit those css vars into skeleton.